client.go

```diff
@@ -27,6 +27,17 @@ type Client struct {
 
 	//Option to specify extra headers like User-Agent
 	ExtraHeader map[string]string
+
+	// RateLimiting is used to store the rate limitting stats.
+	// More information in the official documentation: https://docs.datadoghq.com/api/?lang=bash#rate-limiting
+	RateLimitingStats map[string]RateLimit
+}
+
+type RateLimit struct {
+	limit     string
```

Comment:
all the fields should be public, else you will not be able to use it from another project
```suggestion
	Limit     string
```